test/Schemes/K3.jl

```diff
@@ -0,0 +1,97 @@
+#@testset "A K3 surface from an elliptic fibration" begin
```

Comment:
@afkafkafk13 : This is the test file to check out the bottleneck for the singular locus computation. The full script never finished for me, but one can also run it in parts, say, until the third blowup.

Reply:
Can we discuss about this directly (gather/BBB/phone)?